recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - patch  # [unix]
-    - python >=3
+    - python
   host:
     - openjdk >=8
     - posix  # [win]
```

Comment:
Do we need `libarchive` too as Arch Linux did for 5.3.2 or even >=6.0.0? See https://github.com/archlinux/svntogit-community/blob/1793709708219e7e6fbb8a0c628054323e4439c9/trunk/PKGBUILD#L13

Reply:
It's a good point you bring up, Serhii. Sadly I cannot see where it is being used in bazel, exactly -- this codebase is quite strange to me. The only reference I can find at all is in some Objective C-related Java code, which I don't know if we care about (I suspect we don't?)

This is all I know:
- Arch Linux had it even before 5.3.2, going back to at least 4.0. And we did not.
- conda-forge does not have it.

So, I would opt to keep things the same, and leave it out.